std/assembly/string.ts

```diff
@@ -24,6 +27,86 @@ export class String {
     return out;
   }
 
+  static fromCodePoint(code: i32, code1: i32 = -1, code2: i32 = -1): String {
```

Comment:
For now we haven't variable function arguments (or rest/spread operator). So I implement this as necessary signature with maximum 3 code points for input. This enough for all cases. Later I also add  `fromCodePoints` which handle array as input.

Reply:
I see, thanks. I assume the main advantage of batching here is that just one string must be allocated instead of concatenating multiple, but it also has a cost where `fromCodePoint` is called multiple times with a single codepoint instead, due to asserting the then unused arguments. Wdyt of making the multi-argument variant an internal helper instead and leaving the `fromCodePoint` API with just one parameter for now?

Reply:
Hmm, actually I implemented `fromCodePoint` at the same time for testing string comparison which required 3 arguments. So you propose create `_fromCodePoint(c0: i32, c1: i32, c2: i32)` and 
```ts
fromCodePoint(code: i32): String { 
   return _fromCodePoint(code, -1, -1);
}
```
?

Reply:
When thinking about it a little more it's more some sort of feeling that we should keep those things straight forward for now, because we don't know how variable arguments might look like in the future. Some people might, for example, rely on `fromCodePoint` taking up to three arguments, because they can, even if that might do something entirely different, performance-wise, or even break at some point for whatever reason.

Reply:
ok, done! Now allowing only one argument.